lecture-notes/4-cor.md

```diff
@@ -331,14 +399,36 @@ write some tests.
 
 ```c
 TEST(MySoftwareModule, IsEvenWithOddNumberReturnsFalse) {
-  EXPECT_EQ(isEven(7), false);
+  EXPECT_FALSE(isEven(7));
 }
 
 TEST(MySoftwareModule, IsEvenWithEvenNumberReturnsTrue) {
-  EXPECT_EQ(isEven(8), true);
+  EXPECT_TRUE(isEven(8));
 }
+
+UTEST_MAIN();
+```
+
+and run them:
+
+```console?prompt=$
+$ cc test.c
+$ ./a.out
+[==========] Running 2 test cases.
+[ RUN      ] MySoftwareModule.IsEvenWithOddNumberReturnsFalse
+[       OK ] MySoftwareModule.IsEvenWithOddNumberReturnsFalse (631ns)
+[ RUN      ] MySoftwareModule.IsEvenWithEvenNumberReturnsTrue
+[       OK ] MySoftwareModule.IsEvenWithEvenNumberReturnsTrue (631ns)
+[==========] 2 test cases ran.
+[  PASSED  ] 2 tests.
+$
 ```
 
+Neat. The tests passed. But what does that *mean*?
+
+Ignore the particulars of the syntax and the names of things for a moment.
+Focus on the function calls to `isEven`.
```

Comment:
```suggestion
Focus on the function calls to `isEven()`.
```
I tend to use parens to disambiguation function names from symbol names, but feel free to ignore if that's not your precedent here.